extensions/amp-geo/0.1/amp-geo.js

```diff
@@ -374,8 +364,8 @@ export class AmpGeo extends AMP.BaseElement {
         }
 
         return {
-          ISOCountry: self.country_,
-          matchedISOCountryGroups: self.matchedGroups_,
+          ISOCountry: this.country_,
```

Comment:
Since we move all to JsonObject, do you also want to change this one, so that we don't need to hoist its typedef in amp.extern.js

Reply:
either way works for me. i just didn't want to assume what the author's intent was.